### DISCUSSION TITLE
Use fonts that are easier to read for code

### DISCUSSION THREAD

**Discussion by Val-istar-Guo** [vuejs/pinia#1242] [Ideas]
![image](https://user-images.githubusercontent.com/12405937/164276370-a75ff7e8-ecd9-4793-b1ff-3d5f65117946.png)

the character 's'  and △ are very similar.
This makes reading more difficult

**Comment by anktx** [comment]
merge with #1236, #738

**Comment by Saman-Safaei** [comment]
what is the font name of pinia site codes ?
